packages/@cdktf/hcl2cdk/test/__snapshots__/resources.test.ts.snap

```diff
@@ -1,5 +1,42 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`resources access use fqn for data source typescript snapshot 1`] = `
+"import * as constructs from \\"constructs\\";
+import * as cdktf from \\"cdktf\\";
+/*Provider bindings are generated by running cdktf get.
+See https://cdk.tf/provider-generation for more details.*/
+import * as aws from \\"./.gen/providers/aws\\";
+class MyConvertedCode extends cdktf.TerraformStack {
+  constructor(scope: constructs.Construct, name: string) {
+    super(scope, name);
+    new aws.provider.AwsProvider(this, \\"aws\\", {
+      region: \\"us-east-1\\",
+    });
+    const dataAwsAvailabilityZonesChangemeAzListEbsSnapshot =
+      new aws.dataAwsAvailabilityZones.DataAwsAvailabilityZones(
+        this,
+        \\"changeme_az_list_ebs_snapshot\\",
+        {
+          state: \\"available\\",
+        }
+      );
+    new aws.ebsVolume.EbsVolume(this, \\"changeme_ebs_volume_snapshot\\", {
+      availabilityZone:
+        \\"\${\\" +
+        dataAwsAvailabilityZonesChangemeAzListEbsSnapshot.fqn +
```

Comment:
Is it not possible to get full accessors without the use of fqn?

Reply:
The alternative would be to use propertyAccess, but I thought we decided not to use those too much?